client/src/components/CalendarPage.js

```diff
@@ -58,6 +59,8 @@ class CalendarPage extends React.Component {
   }
 
   getStatus = (l) => {
+    if (l.deleted_at)
+      return "canceled";
     if ((moment(l.datetime).isBefore(moment().format("YYYY-MM-DD"))))
       return "closed"
     if (l.booking_updated_at)
```

Comment:
We can probably define a Lecture status object and return the status by using something like `LectureStatus.Canceled`

Reply:
I totally agree, since we have an ExtendedLecture class already we can put there some methods, so even if we change the actual JSON key names we still have a working code